### DISCUSSION TITLE
Plans to support .NET 6.0?

### DISCUSSION THREAD

**Discussion by wouterroos** [christopherread/Obvs#55] [Q&A] [answered]
Hi,

As stated in the title, are there plans to support .NET 6.0 and if so, what would be the ETA?

Thanks!

**Comment by megakid** [comment, marked as answer]
Definitely - I can try to get something together on this, although there's no real reason why you can't start using Obvs in your .NET 6 projects immediately (unless I'm mistaken here?)

**Reply by wouterroos** [reply]
@megakid Good to hear! I haven't used Obvs in a project as of yet, but we're looking for a proper message broker abstraction for our microservice framework and this might be a good fit.
